CHANGELOG.md

```diff
@@ -8,6 +8,10 @@ Steep 1.2+ is required. Read more about the [feature](https://hackmd.io/xLrYaqUt
 
 - Added `manifest.yml` for RBS. ([@palkan][])
 
+## 2.3.0 (2023-01-11)
```

Comment:
Please, do not change the version in the change log; we can not know when this PR will be released

Reply:
done